src/MainCase/MainCase.scad

```diff
@@ -3,7 +3,7 @@ include <../../variables.scad>
 use <Round-Anything/polyround.scad>
 use <../../lib/MountRails.scad>
 use <../../lib/utils.scad>
-use <./UsbCover.scad>
+use <./UsbCover.scad> 
```

Comment:
Please configure your editor to remove empty whitespace at EOL.

Reply:
I was using OpenSCAD so far for obvious reasons and didn't see that option. Removed this whitespace from the PR though.

Reply:
You can use OpenSCAD in "Automatic reload and preview" mode and then use your normal editor for writing the code ;)